apps/web-giddh/src/app/shared/template-froala/template-froala.component.html

```
@@ -1,171 +1,457 @@
-        <span mat-dialog-close aria-label="close icon"><i class="icon-cross"></i></span>
-    </div>
+    <ng-container
+        appTranslate
+        [file]="'template-froala'"
+        (commonLocaleData)="commonLocaleData = $event"
+        (localeData)="localeData = $event"
+        (translationComplete)="translationComplete($event)"
+    >
+        <div class="dialog-header d-flex align-items-center form-header position-sticky top-0">
+            <h3 mat-dialog-title>{{ commonLocaleData?.app_custom_email_template }}</h3>
+            <button mat-icon-button cdkFocusInitial mat-dialog-close aria-label="close icon"class="p-0"><i class="icon-cross"></i></button>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)